DTML.EduBot/Dialogs/NoneDialog.cs

```diff
@@ -22,10 +22,10 @@ public partial class ChitChatDialog : QnaLuisDialog<object>
             await context.PostAsync(BotPersonality.BotResponseUnrecognizedIntent);
         }
 
-       [LuisIntent("Gibberish")]
-        public async Task HandleGibberish(IDialogContext context, LuisResult result)
-        {
-            await context.SayAsync(BotPersonality.BotResponseToGibberish, BotPersonality.BotResponseToGibberish);
-        }
+       //[LuisIntent("Gibberish")]
```

Comment:
Why commenting out? If not in use, we should delete the code block.

With that said, should this code block still function?